.github/workflows/test.yml

```diff
@@ -7,14 +7,27 @@ jobs:
     name: Test
     runs-on: ubuntu-latest
 
+
     steps:
     - name: Checkout Code
       uses: actions/checkout@v2
+    - name: Hack to get setup-python to work on nektos/act
+      run: |
+        if [ ! -f "/etc/lsb-release" ] ; then
+          echo "DISTRIB_RELEASE=18.04" > /etc/lsb-release
+        fi
     - name: Set up Python 3.7
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
     - name: Install Dependencies
       run: pip install -e '.[all]'
     - name: Run Pytest
-      run: python -m pytest -s -v --runslow
\ No newline at end of file
+      run: |
+        pip install pytest-cov
+        python -m pytest -s -v --cov=./lightly --cov-report=xml --ignore=./lightly/openapi_generated/
```

Comment:
Make sure we also run the **slow** tests as before (see https://github.com/lightly-ai/lightly/blob/develop/.github/workflows/test.yml)